src/components/Auth/Forms/login.tsx

```diff
@@ -58,12 +64,10 @@ const LoginForm = () => {
 
   return (
     <div>
-      <div className="w-full">
-        <SignInOAuthButton
-          strategy="oauth_google"
-          label="Continue with Google"
-          icon={googleIcon}
-        />
+      <div className="relative flex justify-center align-center pt-5 pb-4 gap-x-0.5">
+        <SignInOAuthButton strategy="oauth_facebook" icon={facebookIcon} />
+        <SignInOAuthButton strategy="oauth_google" icon={googleIcon} />
+        <SignInOAuthButton strategy="oauth_microsoft" icon={microsoftIcon} />
```

Comment:
Can't remember accessibility concerns wrt Repair Lab off the top of my head but for buttons, especially those for OAuth sign-ins, ensure that they are accessible. This includes proper labeling (using `aria-label` if necessary) so that screen reader users can understand the purpose of each button.